reconcile/utils/ocm.py

```diff
@@ -25,23 +26,27 @@ class OCM:
     """
     OCM is an instance of OpenShift Cluster Manager.
 
+    :param name: OCM instance name
     :param url: OCM instance URL
     :param access_token_client_id: client-id to get access token
     :param access_token_url: URL to get access token from
     :param offline_token: Long lived offline token used to get access token
     :param init_provision_shards: should initiate provision shards
     :param init_addons: should initiate addons
+    :param blocked_versions: versions to block upgrades for
     :type url: string
     :type access_token_client_id: string
     :type access_token_url: string
     :type offline_token: string
     :type init_provision_shards: bool
     :type init_addons: bool
+    :type blocked_version: list
     """
-    def __init__(self, url, access_token_client_id, access_token_url,
+    def __init__(self, name, url, access_token_client_id, access_token_url,
                  offline_token, init_provision_shards=False,
-                 init_addons=False):
+                 init_addons=False, blocked_versions=None):
```

Comment:
Given how this one is being used later on, can we convert it to a set?

Reply:
it is being passed as a list from app-interface

Reply:
added handling for a set